app/Http/Controllers/SetupController.php

```diff
@@ -51,8 +51,18 @@ public static function displaySetupPage(Request $request) {
         if (env('POLR_SETUP_RAN')) {
             return self::setupAlreadyRan();
         }
-
-        return view('setup');
+        
+        // see if any of the variables have already been provided in .env file
+        // in which case pass those to the view, otherwise assume defaults
+        $view_data = array(
+            'db_host' => (getenv('DB_HOST')) ? getenv('DB_HOST') : 'localhost',
```

Comment:
These can be: `'db_host' => env('DB_HOST', 'localhost')` , bit more concise.

Reply:
Easy enough to switch over. I believe env() is Laravel specific, so I've always used getenv() which is PHP native, but this is a Laravel project and yes that is more concise.